addons/mission/functions/fnc_sleeping.sqf

```diff
@@ -0,0 +1,42 @@
+#include "script_component.hpp"
+/*
+ * Author: Josh, Mike
+ * Sets units in a sleeping animation, alternative call to wake them up.
```

Comment:
Two functions would be better, this is just confusing.

Reply:
If you want the units to be asleep, true if not false. Super simple.

Reply:
Function name and arguments are confusing.

Time is only used if `_asleep = true`, so the wake-up delay. But it could also be read as needing only first call to put them asleep and they will wake-up automatically after that time delay. Maybe this can just be documented better. Or actually add support for single-call auto-wakeup too.

Function name should be more descriptive, something like `setSleeping` would be fitting. Just `sleeping` makes me think of `sleep` way too much.

Also `MFUNC` in example.

Reply:
Changed it's name, added more descriptions to the header.

Although just remember the wiki will have detailed examples for people to use.